src/EnergyPlus/EnergyPlusPgm.cc

```diff
@@ -224,6 +224,11 @@
 #endif
 
 void EnergyPlusPgm(std::string const &filepath)
+{
+    std::exit(EnergyPlusPgmReturnCodes(filepath));
+}
+
+int EnergyPlusPgmReturnCodes(std::string const & filepath)
```

Comment:
I really thought from the PR description that this function was going to give me the possible return codes from E+.  We'll figure out a better name.